App.js

```diff
@@ -1,69 +1,182 @@
-import React, { Component } from 'react';
-import {Text} from "react-native";
-
+import React, { Component } from "react";
 import Workshops from "./screens/Workshops";
-import Home from './screens/Home';
+import Home from "./screens/Home";
 import Schedule from "./screens/Schedule";
-import Event from './screens/Event';
+import Event from "./screens/Event";
+import Login from "./screens/Login";
+import {
+    createBottomTabNavigator,
+    createStackNavigator,
+    createAppContainer
+} from "react-navigation";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
 import {
-  createBottomTabNavigator,
-  createStackNavigator,
-  createAppContainer,
-} from 'react-navigation';
-import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
-import { faHome, faList, faBell } from '@fortawesome/free-solid-svg-icons';
+    faHome,
+    faList,
+    faBell,
+    faKey,
+    faQuestion
+} from "@fortawesome/free-solid-svg-icons";
+import AsyncStorage from "@react-native-community/async-storage";
+import { authorize } from "react-native-app-auth";
+
+export const AuthContext = React.createContext();
+const authUrl = "https://login.hack.gt";
```

Comment:
Seems a bit tedious, make a TODO/open an issue to move to env var
https://stackoverflow.com/questions/33117227/setting-environment-variable-in-react-native